infrastructure/proto/log_replication_cluster_info.proto

```diff
@@ -29,8 +29,8 @@ message NodeConfigurationMsg {
 
 enum ClusterRole {
   INVALID = 0;
-  ACTIVE = 1;
-  STANDBY = 2;
+  SOURCE = 1;
```

Comment:
thinking if it would be better to mark ACTIVE/STANDBY as deprecated and add the new values instead...

Reply:
I am guessing for rolling upgrade?
If we do that, we'll need to also preserve the active/standby checks in the code.

Reply:
I think renaming Enums without modifying their field numbering should not break compatibility.

Reply:
this was the article I came across - https://stackoverflow.com/questions/50857940/renaming-enum-value-in-protobuf-backward-compatibility

If we only use the binary format for protobufs, it will be compatible when doing rolling upgrade.  IMO, this is the case for us.